plugins/jobs/server/models/job.py

```diff
@@ -264,7 +264,7 @@ def createJob(self, title, type, args=(), kwargs=None, user=None, when=None,
             'log': [],
             'meta': {},
             'handler': handler,
-            'async': async,
```

Comment:
What do @girder/developers think about changing the schema here? It is more consistent, but since Girder currently lacks any notion of database migrations (#946) - this could cause issues in the future for existing sites.

Reply:
I think that the move to Girder 3 is a fine time to change this.  We could use `job.get('asynq', job.get('async'))` to read the old `async` key if no `asynq` key exists.

I'd mildly prefer `async_` to `asynq`, as I suspect I will always mistype `asynq`.

Reply:
Do we want to switch it over to async_ ? Or would another convention be better